esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -687,7 +687,8 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
                 features.appendf("%s%s:%s", (index++ == 0 ? "" : ", "), key, getSecAccessFlagName(*methaccessmap.getValue(key)));
             }
             const char * user = context.queryUserId();
-            throw MakeStringException(-1, "%s::%s access denied for user '%s' - Method requires: '%s'.", srvdef.queryName(), mthName, (user && *user) ? user : "Anonymous", features.str());
+            throw MakeStringException(401, "Insufficient priviledge to run function (%s) access denied for user (%s) - %s", mthName, (user && *user) ? user : "Anonymous", features.str());
+
```

Comment:
Should we match the scapps message exactly? If so the message returned by scapps is shorter, it just says "Insufficient privilege to run function %s".  Note the spelling of 'privilege' and no parens around the method name.

Reply:
The extra bit of information could be useful, but I could make it mostly the same.  features.str() also has the required level.

AllowXyz:Full vs AllowXSyz